osu.Game/Overlays/Changelog/UpdateStreamBadge.cs

```diff
@@ -16,99 +14,90 @@
 using osu.Game.Graphics.Sprites;
 using osu.Game.Graphics.UserInterface;
 using osuTK;
-using osuTK.Graphics;
 
 namespace osu.Game.Overlays.Changelog
 {
     public class UpdateStreamBadge : TabItem<APIUpdateStream>
     {
-        private const float badge_height = 66.5f;
         private const float badge_width = 100;
         private const float transition_duration = 100;
 
-        private readonly ExpandingBar expandingBar;
-        private SampleChannel sampleClick;
-        private SampleChannel sampleHover;
-
-        private readonly FillFlowContainer<SpriteText> text;
-
         public readonly Bindable<APIUpdateStream> SelectedTab = new Bindable<APIUpdateStream>();
 
-        private readonly Container fadeContainer;
+        private readonly APIUpdateStream stream;
+
+        private Container fadeContainer;
+        private FillFlowContainer<SpriteText> text;
+        private ExpandingBar expandingBar;
 
         public UpdateStreamBadge(APIUpdateStream stream)
             : base(stream)
         {
-            Size = new Vector2(stream.IsFeatured ? badge_width * 2 : badge_width, badge_height);
+            this.stream = stream;
+        }
+
+        [BackgroundDependencyLoader]
+        private void load(OverlayColourProvider colourProvider)
+        {
+            Size = new Vector2(stream.IsFeatured ? badge_width * 2 : badge_width, 60);
```

Comment:
The height was changed from 66.5 here, and web-side it is 66.5 right now, but it was not a set pixel size as far as I can tell, it just came out like this due to sizings and `line-height: 1.25;`. We could add paddings to emulate line height but unsure if it matters that much.